packages/block-editor/src/components/list-view/block.js

```diff
@@ -106,6 +106,7 @@ function ListViewBlock( {
 	const selectEditorBlock = useCallback(
 		( event ) => {
 			selectBlock( event, clientId );
+			event.preventDefault();
```

Comment:
Had to add this because if focus was placed on a button, the button would get clicked causing an unprovoked action. E.g.
1. Insert a Table block.
2. Select it from the List View.
3. The Create Table button gets clicked on block selection button click.
4. Adding this stops the event after selection.